Lumen Edge Private Cloud/General/cpc-metrics.md

```diff
@@ -1,6 +1,6 @@
 {{{
   "title": "CPC Metrics",
-  "date": "6-15-2020",
+  "date": "06-15-2020",
```

Comment:
This date is incorrect.

Reply:
I may have put that date on, but that article is in a different branch. The date change is NOT SUPPOSED to be part of this pull request. I'll do a different pull request for the entire article once I do pull requests for all the images in the article.

Reply:
Sorry for the confusion -- please delete this pull request.